### DISCUSSION TITLE
Osu lazer folder moved can't retrieve data

### DISCUSSION THREAD

**Discussion by AlanKovksy** [ppy/osu#28508] [Q&A] [answered]
I moved my osu folders from C: appdata roaming and local to my D: drive because i needed space
Doing so my Osu lazer have no data, so with the wizard I want to select my new location with my old files

But I can't select the folder I moved to D: to be the location, it only shows whats inside.
I can though select an old osu stable save but....

If someone could help me that would be super cool

**Comment by BlockHead007** [comment, marked as answer]
If I got it correct, then you have already moved game data to different location but for some reason game didn't remember its location. You should be able to select it manually just by navigating to it and pressing Select directory when you in it:
![image](https://github.com/ppy/osu/assets/25553854/e2ae157f-bb5c-415d-a64b-fa02729a6e55)

**Reply by AlanKovksy** [reply]
omg i thought I already did that, but apparently no and it worked ! thank you so much you saved me !
